src/data/team.json

```diff
@@ -5,62 +5,62 @@
     "role": "Team lead",
     "photoUrl": "olga-tiutiunnyk",
     "githubUrl": "https://github.com/izachok",
-    "text": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum vitae saepe odio consectetur, optio doloribus, soluta reiciendis officiis consequatur qui molestiae tempore expedita."
+    "text": "Olga is our Teamlead. She prepared structure of project, wrote documents about project interfaces and made many reviews. She implemented footer and this modal about team with slider, wrote functions for working with API, refactored project in order to connect all parts together. Add to library buttons also were implemented by her. "
```

Comment:
Olga is our Team Leader. She prepared the structure of the project, wrote documentation about the project's interfaces, and made most of the reviews. She implemented the footer and the modal section with a slider about our team, also wrote functions for working with API, and refactored the project to connect all the parts. "Add to Library" buttons are also made by Olga.